ca_gazebo/launch/prueba.launch

```diff
@@ -0,0 +1,29 @@
+<launch>
+	<arg name="env" default="prueba"/>
+
+  	<arg name="paused" value="$(optenv PAUSED false)"/>
+	<arg name="gui" value="$(optenv GUI true)"/>
+	<arg name="debug" value="$(optenv DEBUG false)"/>
+	
+	<env name="GAZEBO_MODEL_PATH" value="$(find ca_gazebo)/models"/>
+
+	<!-- Launch Gazebo world -->
+	<include file="$(find gazebo_ros)/launch/empty_world.launch">
+		<arg name="debug" value="$(arg debug)" />
+		<arg name="gui" value="$(arg gui)" />
+		<arg name="paused" value="$(arg paused)"/>
+		<arg name="use_sim_time" value="true"/>
+		<arg name="verbose" value="true"/>
+		<arg name="world_name" value="$(find ca_gazebo)/worlds/$(arg env).world"/>
+	</include>
+
+	<!-- Spawn robot/s -->
```

Comment:
Launch files use [this](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_gazebo/launch/create_house.launch#L10-L12) instead of what you're doing.

Reply:
This is an unnecessary file. Deleted it.